hack/install.sh

```diff
@@ -140,7 +140,7 @@ do_install() {
 			exit 0
 			;;
 
-		'opensuse project'|opensuse|'suse linux'|sled)
+		'opensuse project'|opensuse|'suse linux'|sle[d,s])
```

Comment:
The comma here is superfluous if the goal is just to match both `sles` and `sled` (as-is, this will also match `sle,`). :+1: